src/support/rgw_support_bundle

```diff
@@ -28,6 +28,9 @@ import argparse
 from cortx.utils.errors import BaseError
 from cortx.utils.process import SimpleProcess
 from cortx.utils.conf_store.conf_store import Conf, MappedConf
+from src.const import (
+    CONFIG_PATH_KEY, LOG_PATH_KEY, COMPONENT_NAME,
+    RGW_CONF_FILE, RGW_CRASHDUMP_DIR)
```

Comment:
in any new code, lets avoid using RGW as direct name. like here just use CONF_FILE, CRASHDUMP_DIR
same for code everywhere else